### PR TITLE
opi for FMR kit

### PR DESCRIPTION
**Pull request by rerpha**
### Description of work

*Add your own description here*

### Ticket

*Link to Ticket*

### Acceptance criteria

*List the acceptance criteria for the PR. The aim is provide information to help the reviewer*

### Unit tests

*Give an overview of unit tests you have added or modified, if applicable. The aim is provide information to help the reviewer*

### System tests

*Mention any automated tests or manual tests that you have added or modified, if applicable. The aim is provide information to help the reviewer*

### Documentation
*Highlight and provide a link to any additions or changes to the documentation, if applicable. The aim is provide information to help the reviewer*

---

#### Code Review

- [ ] Is the code of an acceptable quality?
- [ ] Do the changes function as described and is it robust?
- [ ] Is there associated PR for the [release notes](https://github.com/ISISComputingGroup/IBEX/blob/master/release_notes/ReleaseNotes_Upcoming.md)?

### Final Steps
- [ ] Reviewer has also merged the [release notes](https://github.com/ISISComputingGroup/IBEX/blob/master/release_notes/ReleaseNotes_Upcoming.md)

### CODE REVIEW

File: base/uk.ac.stfc.isis.ibex.opis/resources/fmr.opi [hunk 1]
```diff
@@ -0,0 +1,3602 @@
...
+      <name>Label_2</name>
+      <rules />
+      <scale_options>
+        <width_scalable>true</width_scalable>
+        <height_scalable>true</height_scalable>
+        <keep_wh_ratio>false</keep_wh_ratio>
+      </scale_options>
+      <scripts />
+      <show_scrollbar>false</show_scrollbar>
+      <text>Current difference:</text>
```
Comment (JackEAllen) [inline base/uk.ac.stfc.isis.ibex.opis/resources/fmr.opi:3011]: The size of the label box for this is too small for me, does this only display "Current" from inside the client and also from the OPI editor for you too or the entire name "Current difference:"? I'm not sure if it's just a screen resolution problem or not

**Review by JackEAllen — CHANGES_REQUESTED** [review]
From an initial scan through changes made, the overall layout looks good and resembled the VI! 👍  
Going forward, I would like the see each property having a more descriptive label and a few other minor changes to allow for all tests to pass when executing the `check_opi_format.py` script. 
Functionally, I haven't ran the VI yet to make sure value are correctly represented in the OPI so will come back to this and update if all works well or if any changes are needed.

File: base/uk.ac.stfc.isis.ibex.opis/resources/fmr.opi [hunk 2]
```diff
@@ -0,0 +1,3602 @@
...
+      <name>Label_4</name>
+      <rules />
+      <scale_options>
+        <width_scalable>true</width_scalable>
+        <height_scalable>true</height_scalable>
+        <keep_wh_ratio>false</keep_wh_ratio>
+      </scale_options>
+      <scripts />
+      <show_scrollbar>false</show_scrollbar>
+      <text>VI activity:</text>
```
[the remark this answers is not in the document]
Comment (rerpha) [reply base/uk.ac.stfc.isis.ibex.opis/resources/fmr.opi]: It'll still fail but the units are Ba so we can't do much about that - doesn't look like there's a way to skip it so we might just need to leave it failing - some other OPIs are the same i believe

File: base/uk.ac.stfc.isis.ibex.opis/resources/fmr.opi (hunk 1 above)
Comment (rerpha) [reply base/uk.ac.stfc.isis.ibex.opis/resources/fmr.opi:3011]: It shows the whole thing for me - are you on a 4k screen? I don't think the GUI likes scaling to be anything other than 100% either

Comment (JackEAllen) [reply base/uk.ac.stfc.isis.ibex.opis/resources/fmr.opi:3011]: I'm on 1920*1200

Comment (rerpha) [reply base/uk.ac.stfc.isis.ibex.opis/resources/fmr.opi:3011]: hmm, weird, not sure then. looks ok to me even when minimised with the same screen res

**Pull request merged**